doc/R-package/index.rst

```diff
@@ -9,22 +9,28 @@ XGBoost R Package
 
 You have found the XGBoost R Package!
 
-***********
-Get Started
-***********
-* Checkout the :doc:`Installation Guide </install>` contains instructions to install xgboost, and :doc:`Tutorials </tutorials/index>` for examples on how to use XGBoost for various tasks.
-* Read the `API documentation <https://cran.r-project.org/web/packages/xgboost/xgboost.pdf>`_.
+***************
+Getting Started
+***************
+
+- Checkout the :doc:`Installation Guide </install>` contains instructions to install xgboost, and :doc:`Tutorials </tutorials/index>` for examples on how to use XGBoost for various tasks.
+- `XGBoost R tutorials and API reference <../r_docs/index.html>`__
+- `CRAN document <https://cran.r-project.org/web/packages/xgboost/xgboost.pdf>`__
 
 *********
-Tutorials
+Vignettes
```

Comment:
Here it could try to embed the .md if it's somehow possible to generate them with the Makefile.

Reply:
Done.